lib/main.dart

```diff
@@ -1,25 +1,19 @@
 import 'package:flutter/material.dart';
-import 'Home.dart';
+import 'package:firebase_auth/firebase_auth.dart';
```

Comment:
Let's remove all instances of the firebase_auth, since we won't be using firebase auth anymore.